src/libs/API/parameters/TransactionResolveParams.ts

```diff
@@ -0,0 +1,17 @@
+type TransactionResolveParams = {
+    transactionID: string;
+    transactionIDList: string[];
```

Comment:
I would add some comments to these two to explain one of is the transaction we would like to keep while we will be putting the others on hold

Reply:
I updated.